src/Microsoft.Sbom.Api/Microsoft.Sbom.Api.csproj

```diff
@@ -53,6 +53,7 @@
       <ProjectReference Include="..\Microsoft.Sbom.Contracts\Microsoft.Sbom.Contracts.csproj" />
       <ProjectReference Include="..\Microsoft.Sbom.Extensions\Microsoft.Sbom.Extensions.csproj" />
       <ProjectReference Include="..\Microsoft.Sbom.Parsers.Spdx22SbomParser\Microsoft.Sbom.Parsers.Spdx22SbomParser.csproj" />
+      <ProjectReference Include="..\Microsoft.Sbom.Parsers.Spdx30SbomParser\Microsoft.Sbom.Parsers.Spdx30SbomParser.csproj" />
```

Comment:
We should file an issue that our encapsulation is broken here. I doubt we'll ever do anything about it, but when we need a future breaking change, maybe we can pick this up. Simply put, we should be using dependency injection to maintain loose coupling here